src/types/entityTypes.ts

```diff
@@ -20,3 +20,70 @@ export interface EntityContext<T> extends InvocationContext {
      */
     df: DurableEntityContext<T>;
 }
+
+/**
+ * An entity context with dummy default values to facilitate mocking/stubbing the
+ * Durable Functions API.
+ */
+export declare class DummyEntityContext<T> extends InvocationContext implements EntityContext<T> {
+    /**
+     * Creates a new instance of a dummy orchestration context.
```

Comment:
Looks like you copied the `orchestration` comments, not the `entity` comments

Reply:
Oops thanks, fixed!